frontend/src/routes/_dashboard/create-project.lazy.tsx

```diff
@@ -0,0 +1,72 @@
+import * as Form from "@radix-ui/react-form";
+import { createFileRoute } from "@tanstack/react-router";
+import { withAuthRedirect } from "~/components/helper/auth-redirect";
+import { useAuthUser } from "~/components/helper/use-auth-user";
+import { MetaTitle } from "~/components/meta-title";
+import { Button } from "~/components/ui/button";
+import { Input } from "~/components/ui/input";
+import { Textarea } from "~/components/ui/textarea";
+
+export const Route = createFileRoute("/_dashboard/create-project")({
+  component: withAuthRedirect(AuthedView)
+});
+
+function AuthedView() {
+  const query = useAuthUser();
+  const user = query.data?.data?.user;
+
+  if (!user) {
+    return null;
+  }
+
+  return (
+    <dl>
+      <h1>
+        <MetaTitle title="Create Project" />
+        <CreateProject />
+      </h1>
+    </dl>
```

Comment:
```suggestion
    <>
        <MetaTitle title="Create Project" />
        <CreateProject />
    </>
```

What is this markup ?  This is totally unnecessary.

I noticed it is the same on the `index.lazy` file, you need to remove all the markup outside

Reply:
You suggest this approach long time ago ...